[2024-07-04T01:42:28.126+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-04T01:42:28.200+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-21T00:00:00+00:00 [queued]>
[2024-07-04T01:42:28.223+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-21T00:00:00+00:00 [queued]>
[2024-07-04T01:42:28.225+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-07-04T01:42:28.260+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): run_script> on 2024-03-21 00:00:00+00:00
[2024-07-04T01:42:28.287+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=482) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-04T01:42:28.291+0000] {standard_task_runner.py:63} INFO - Started process 499 to run task
[2024-07-04T01:42:28.292+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'exchange_rate_dag', 'run_script', 'scheduled__2024-03-21T00:00:00+00:00', '--job-id', '39', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpp4y_18p_']
[2024-07-04T01:42:28.294+0000] {standard_task_runner.py:91} INFO - Job 39: Subtask run_script
[2024-07-04T01:42:28.407+0000] {task_command.py:426} INFO - Running <TaskInstance: exchange_rate_dag.run_script scheduled__2024-03-21T00:00:00+00:00 [running]> on host e26a552f405a
[2024-07-04T01:42:28.632+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juan_ml' AIRFLOW_CTX_DAG_ID='exchange_rate_dag' AIRFLOW_CTX_TASK_ID='run_script' AIRFLOW_CTX_EXECUTION_DATE='2024-03-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-21T00:00:00+00:00'
[2024-07-04T01:42:28.635+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-04T01:42:28.637+0000] {logging_mixin.py:188} INFO - All credentials are set properly.
[2024-07-04T01:42:28.638+0000] {logging_mixin.py:188} INFO - Error al crear la tabla en Redshift: invalid literal for int() with base 10: 'None'
[2024-07-04T01:42:29.751+0000] {logging_mixin.py:188} INFO - Error al realizar la solicitud a la API: 401 Client Error: Unauthorized for url: https://api.apilayer.com/exchangerates_data/latest?symbols=EUR,USD,CNY&base=ARS
[2024-07-04T01:42:29.759+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-04T01:42:29.763+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-04T01:42:29.792+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=exchange_rate_dag, task_id=run_script, run_id=scheduled__2024-03-21T00:00:00+00:00, execution_date=20240321T000000, start_date=20240704T014228, end_date=20240704T014229
[2024-07-04T01:42:29.880+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-04T01:42:29.917+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-07-04T01:42:29.921+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
